src/Controller/EnMarche/CitizenProjectController.php

```diff
@@ -45,4 +49,35 @@ public function helpAction(): Response
 
         return new Response();
     }
+
+    /**
+     * @Route("/skills/autocompletion",
+     *     name="app_citizen_project_skills_autocomplete",
+     *     condition="request.isXmlHttpRequest()"
```

Comment:
Please add `and request.query.get("category") and request.query.get("term")` to prevent useless queries, the javascript itself should not trigger an Ajax request with empty params.